ngraph/run.sh

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
+
+KUBEFLOW_DIR=$1
+TAG=$2
+GCLOUD_PROJECT=$3
+PID=''
+PORT=6006
+
+portforward() {
+  local pod=$1 namespace=$2 from_port=$3 to_port=$4 cmd
+  kubectl port-forward $pod ${from_port}:${to_port} --namespace=$namespace 2>&1>/dev/null 
+}
+
+waitforpod() {
+  local cmd="kubectl get pods --no-headers --selector=job-name --field-selector=status.phase=Running --namespace=kubeflow 2>/dev/null" found=$(eval "$cmd") finished
+  while [[ -z $found ]]; do
+    sleep 1
+    found=$(eval "$cmd")
+  done
+  found=${found%% *}
+  cmd="kubectl logs $found | grep '^TensorBoard 1.12.0'"
+  finished=$(eval "$cmd")
+  while [[ -z $finished ]]; do
+    sleep 1
+    finished=$(eval "$cmd")
+  done
+  echo $found
+}
+
+echo 'Deploying kubeflow...'
+$KUBEFLOW_DIR/scripts/kfctl.sh init ngraph --platform none
+cd ngraph
+$KUBEFLOW_DIR/scripts/kfctl.sh generate all
+cd ks_app/
+ks pkg install kubeflow/jobs
```

Comment:
As I mentioned in your other PR why create a ksonnet package for a generic K8s job?
Why not just define a ksonnet app for the application with whatever manifests you need and then just tell users to add an environment to that ksonnet app?

Reply:
Great thanks.